### PR TITLE
Misc doc suggestions

### PR DESCRIPTION
**Pull request by rocky**
Some small things noticed. 

When there is only one Example, I find a section heading Examples misleading. (When there is more than one which I hope eventually happens, then a plural is justified.

### CODE REVIEW

File: docs/types/color.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-It's an array of 3 floats from 0 to 1 (red, green and blue).
+A color is an array of 3 floats, for the red, green and blue value. A value should be in the range 0 to 1.
 
 To convert a 0-255 number to 0-1 you just need to divide 1 by the number.
```
Comment (rocky) [inline docs/types/color.md:3]: I don't believe this comment is correct the number is divided by 255, not 1.

Comment (TiagoCavalcante) [reply docs/types/color.md:3]: In WebGL the colors are in this range.

**Comment by TiagoCavalcante** [comment]
The 2 spaces in the end are line breaks.

**Comment by TiagoCavalcante** [comment]
Rocky I'll add more examples to the sections with only 1.

**Pull request merged**